<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogSetpointVibePattern</class>
 <widget class="QDialog" name="QDialogSetpointVibePattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Vibration pattern setpoint</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_state">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select device state</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_on">
        <property name="text">
         <string>ON</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_off">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_select">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_periods">
      <property name="text">
       <string>Vibration periods:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_periods">
      <property name="text">
       <string>1000, 100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_period_limits">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Minimum value is 100 ms.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_periods">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_frequencies">
      <property name="text">
       <string>Vibration frequencies:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_frequencies">
      <property name="text">
       <string>440, 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_frequency_limits">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Allowed frequency range is [1,1000] Hz.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_frequencies">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_amplitudes">
      <property name="text">
       <string>Vibration amplitudes:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_amplitudes">
      <property name="text">
       <string>50, 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_range_limits">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Allowed amplitude range is [0,50] %.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_amplitudes">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>label_state</zorder>
  <zorder>radioButton_on</zorder>
  <zorder>radioButton_off</zorder>
  <zorder>label_periods</zorder>
  <zorder>label_frequencies</zorder>
  <zorder>label_amplitudes</zorder>
  <zorder>label_period_limits</zorder>
  <zorder>label_frequency_limits</zorder>
  <zorder>label_range_limits</zorder>
  <zorder>lineEdit_periods</zorder>
  <zorder>lineEdit_frequencies</zorder>
  <zorder>lineEdit_amplitudes</zorder>
  <zorder>verticalSpacer_periods</zorder>
  <zorder>verticalSpacer_select</zorder>
  <zorder>verticalSpacer_frequencies</zorder>
  <zorder>verticalSpacer_amplitudes</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogSetpointVibePattern</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogSetpointVibePattern</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
